python/ray/llm/_internal/serve/config_generator/utils/models.py

```diff
@@ -0,0 +1,36 @@
+from typing import Literal, Optional
+
+from pydantic import BaseModel, Field
+
+from ray.llm._internal.serve.config_generator.utils.gpu import GPUType
+
+
+TEXT_COMPLETION_MODEL_TYPE: Literal["TextCompletion"] = "TextCompletion"
+
+ModelType = Literal["TextCompletion"]
+
+
+class ServeModel(BaseModel):
+    id: str
+    hf_token: Optional[str] = None
+    remote_storage_uri: Optional[str] = None
+    gpu_type: GPUType
+
+
+class TextCompletionLoraModelConfig(BaseModel):
+    max_num_lora_per_replica: int
+    uri: Optional[str] = Field(
+        None,
+        description="If not provided, we default to `$ANYSCALE_ARTIFACT_STORAGE/lora_fine_tuning`",
```

Comment:
```suggestion
        description="If not provided, we default to `s3://my-bucket/lora_fine_tuning`",
```
But maybe better to make a meaningful default 🙃